sensorserviceutils/sensorserviceutils.go

```diff
@@ -450,17 +486,39 @@ func (p parser) allSensorInfo() *sipb.AllSensorsInfo {
 	}
 	sort.Sort(allSensors)
 
+	allApp := make(appSlice, 0, len(p.apps))
```

Comment:
Is the TODO a tline 469 still a TODO?

Reply:
No, the feature has been implemented. I will update the comment.